### PR TITLE
Replacing master with main

### PR DESCRIPTION
**Pull request by nickrandolph**
### Description of Change ###
Fixing a bunch of broken links/references due to renaming master branch to main

### Issues Resolved ### 
Link to Contributing docs was broken (ie https://github.com/xamarin/Xamarin.Forms/blob/master/.github/CONTRIBUTING.md doesn't resolve).

### API Changes ###
None

### Platforms Affected ### 
N/A

### Behavioral/Visual Changes ###
None

### Before/After Screenshots ### 
Not applicable

### Testing Procedure ###
Look for anything that might still be broken due to renaming master to main

### PR Checklist ###

- [x] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Check out [A Beginner's Guide for Contributing to Xamarin.Forms](https://devblog
...
 
-Follow the style used by the [.NET Foundation](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md), with two primary exceptions:
+Follow the style used by the [.NET Foundation](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md), with two primary exceptions:
```
Comment (StephaneDelcroix) [inline .github/CONTRIBUTING.md:21]: please revert this one

File: README.md [hunk 2]
```diff
@@ -143,15 +143,15 @@ You should now be able to run any of the UWP UI Tests.
 
 ## Coding Style ##
 
-We follow the style used by the [.NET Foundation](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md), with a few exceptions:
+We follow the style used by the [.NET Foundation](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md), with a few exceptions:
```
Comment (StephaneDelcroix) [inline README.md:146]: please revert

File: Xamarin.Forms.Xaml.UnitTests/MSBuild/MSBuildTests.cs [hunk 3]
```diff
@@ -391,7 +391,7 @@ public void LinkedFile ([Values (false, true)] bool sdkStyle)
 			AssertExists (IOPath.Combine (intermediateDirectory, "XamlC.stamp"));
 		}
 
-		//https://github.com/dotnet/project-system/blob/master/docs/design-time-builds.md
+		//https://github.com/dotnet/project-system/blob/main/docs/design-time-builds.md
```
Comment (StephaneDelcroix) [inline Xamarin.Forms.Xaml.UnitTests/MSBuild/MSBuildTests.cs:394]: please revert

File: build/scripts/generate-docs.ps1 [hunk 4]
```diff
@@ -1,11 +1,11 @@
 param(
-    [string]$branch = "master",
+    [string]$branch = "main",
```
Comment (StephaneDelcroix) [inline build/scripts/generate-docs.ps1:2]: default branch name on the docs repository is still `master`, please revert this

File: build/scripts/generate-docs.ps1 [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
     [string]$token
 )
 
 # Yes, this is ignoring the parameter passed in by VSTS; we don't need the 
 # parameter, but we can't remove it until all the active branches don't need it
-$branch = "master"
+$branch = "main"
```
Comment (StephaneDelcroix) [inline build/scripts/generate-docs.ps1:8]: please revert

**Comment by nickrandolph** [comment]
@StephaneDelcroix thanks for following up on this one - sorry I missed your comments earlier

**Comment by StephaneDelcroix** [comment]
@nickrandolph I messed up with your branch, sorry

do you mind fixing it ? or I have a local copy, with fixes, and I can create a new PR

**Comment by StephaneDelcroix** [comment]
(I'll have to open a new one, I can't reopen this, and I don't get why it was closed)

**Comment by StephaneDelcroix** [comment]
see #11240

**Comment by nickrandolph** [comment]
Oh thanks - I was just going to push a new remote. Oh, wait, so do you need me to do anything?

**Comment by StephaneDelcroix** [comment]
> Oh, wait, so do you need me to do anything?

no, it's all good, thank you

**Pull request closed without merging**